src/config/dynamic-theme-fixes.config

```diff
@@ -880,6 +880,25 @@ polyline
 
 ================================
 
+askjohnmackay.com
+
+INVERT
+[src$="ask-john-mackay-logo.png"]
+[src$="side-banner-05.gif"]
+[src$="reasonwhy.jpg"]
+
+CSS
+body,
+header,
+#site-footer,
+#live-search {
+    background-image: none !important;
```

Comment:
e.g. for the body, is necessary to have it reset the colors? Removing the background image seems good for me.

Reply:
44f366d1e439f59c047b9db1a97e824c04b95c82